src/prefect/orion/orchestration/core_policy.py

```diff
@@ -432,9 +445,11 @@ async def before_transition(
         if not scheduled_time:
             return
 
-        # At this moment, we take the floor of the actual delay as the API schema
+        # At this moment, we round delay to the nearest second as the API schema
         # specifies an integer return value.
```

Comment:
! Are we losing an important part of the jitter here? Should we update the API to return a fl.oat? The client can sleep on float timescales.

Reply:
I think we should but I don't think it's critical if we'd like to discuss this more